packages/main/test/specs/MultiInput.spec.js

```diff
@@ -213,21 +213,38 @@ describe("Keyboard handling", () => {
 		const innerInput =  await input.shadow$("input");
 		const firstToken = await browser.$("#basic-overflow ui5-token:first-child");
 		const lastToken = await browser.$("#basic-overflow ui5-token:last-child");
+		let caretPosition;
 
 		await innerInput.click();
 		await innerInput.keys("ArrowLeft");
 		assert.strictEqual(await lastToken.getProperty("focused"), true, "The last token is focused");
 
 		await innerInput.keys("Home");
```

Comment:
I didn't find any test, when we have text value in the input field and the HOME key is pressed. I think, if such check is missing, we should add one. We have to check, that the caret is moved at the beginning of the user input.

Reply:
Done